api/src/test/java/io/strimzi/api/kafka/model/KafkaUserCrdIT.java

```diff
@@ -43,15 +39,6 @@ void testKafkaUserWithExtraProperty() {
         createDelete(KafkaUser.class, "KafkaUser-with-extra-property.yaml");
     }
 
-    @Test
-    void testKafkaUserWithMissingRequired() {
-        try {
-            createDelete(KafkaUser.class, "KafkaUser-with-missing-required.yaml");
```

Comment:
Do we use `KafkaUser-with-missing-required.yaml` somewhere else as well? Or should we remove it?

Reply:
Good point, I will check and remove it if not.